tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/createMatter.controller.js

```diff
@@ -875,7 +892,7 @@
                         makePrevOrNextButton();
                     }
                 }
-                else if (sectionName == "snCreateAndShare" && cm.iCurrentPage !== 3) {
+                else if (sectionName == "snCreateAndShare" && cm.iCurrentPage !== 3 && cm.inputs.length == 0) {

```

Comment:
cm.inputs.length check to confirm that extra fields are exists of specific term and extra tab is visible or not so we can validate proper tab content after clicking on next button while creating matter.